doc/cla/corporate/medoo.md

```diff
@@ -0,0 +1,15 @@
+China, 2022-04-05
+
+Medoo Tech agrees to the terms of the Odoo Corporate Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+jlzhou zh@medoo.tech https://github.com/medoo
+
+List of contributors:
+
+jlzhou zh@medoo.tech https://github.com/medoo
```

Comment:
I think on those 2 lines the `<login>` part should be your own login `https://github.com/jlzhou` and not the medoo company page?

Reply:
Why make a new PR instead of update this one?

Reply:
To reduce redundant commits in main branch.